packages/core/src/components/TextField/TextField.tsx

```diff
@@ -118,7 +116,6 @@ const TextField: VibeComponent<TextFieldProps, unknown> & {
       inputAriaLabel,
       searchResultsContainerId = "",
       activeDescendant = "",
-      iconsNames = EMPTY_OBJECT,
```

Comment:
Is this a prop? What is this for?

Reply:
If it is and is redundant it should be also documented, and make sure it doesn't have UX implications